src/chrome/content/rules/Yandex.com.xml

```diff
@@ -30,194 +12,40 @@ Fetch error: http://www.weather.yandex.com/ => https://www.weather.yandex.com/:
 		- interactive-answers ²
 		- m.internet ¹
 		- islands ³
-		- ir *
+		- ir ¹
 		- mc ¹
-
 		- news ¹
 		- m.news ¹
 		- pda.news ¹
-
-		- shad ⁵
-		- (www.)?store ¹
-		- wdgt ⁶
-		- xmlsearch ⁵
+		- shad ³
+		- store ¹
+		- www.store ¹
+		- wdgt ⁴
+		- xmlsearch ³
 		- yabs ¹
 
 	¹ Dropped
 	² 404
 	³ Redirects to http
-	* Shareholder
-	⁵ Redirects to http
-	⁶ 403
+	⁴ 403
 
```

Comment:
You can try `www.foo.bar.invalidsubdomain.yandex.com` which does not work over HTTPS, but it makes no sense to list all possible variants.